democracy_club/templates/base.html

```diff
@@ -10,25 +10,29 @@
       {% stylesheet 'styles' %}
     {% endblock extra_site_css %}
     {% block site_meta %}
-      <link rel="shortcut icon" href="{%  static 'images/logo_icon.png' %}">
+      <link rel="shortcut icon" href="{% static 'images/logo_icon.png' %}">
       <link rel="alternate" type="application/atom+xml" title="Democracy Club Blog" href="{% url "hermes_post_feed" %}" />
       <link rel="manifest" href="{% static "manifest.json" %}">
-      <meta property="fb:pages" content="316731041785" />
     {% endblock site_meta %}
+    {% block base_og_tags %} 
+      <meta property="og:description" content="{% block og_description_content %}We build digital tools to support everyone’s participation in UK democracy.{% endblock og_description_content %}" />
+      <meta name="og:image" content="{% block og_image_content %}{{ CANONICAL_URL }}{% static 'images/maze_illustration_no_text.png' %}{% endblock og_image_content %}" />
+      <meta property="og:title" content="{% block og_title_content %}{{ SITE_TITLE }}{% endblock og_title_content %}" />
+      <meta property="og:type" content="{% block og_type_content %}website{% endblock og_type_content %}">
+      <meta property="og:url" content="{{ CANONICAL_URL }}">
 
-    <meta property="og:description" content="{% block og_description_content %}We build digital tools to support everyone’s participation in UK democracy.{% endblock og_description_content %}" />
-    <meta property="og:image" content="{% block og_image_content %}{% static 'images/logo_icon.png' %}{% endblock og_image_content %}" />
-    <meta property="og:title" content="{% block og_title_content %}{{ site_title }}{% endblock og_title_content %}" />
-    <meta property="og:type" content="{% block og_type_content %}website{% endblock og_type_content %}">
-
-
-    {% block base_og_tags %}
-      <meta property="fb:app_id" content="262795130596272" />
-      <meta property="og:url" content="https://democracyclub.org.uk{{ request.path }}" />
-      <!-- Twitter -->
-      <meta property="twitter:card" content="summary_large_image">
-      <meta property="twitter:url" content="https://democracyclub.org.uk{{ request.path }}>
     {% endblock base_og_tags %}
+    <!-- Facebook -->
+    <meta property="fb:pages" content="316731041785" />
+    <meta property="fb:app_id" content="262795130596272" />
+    <!-- Twitter -->
+    <meta name="twitter:title" content="{{ SITE_TITLE }}">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@democlub">
+    <meta name="twitter:url" content="{{ CANONICAL_URL }}">
```

Comment:
Should this use a `block` so that the url can be different on the blog pages? Otherwise the link will always be to the homepage won't it?

Reply:
Checked this out locally and tested this by visiting a blog page, then inspecting the page source, and here is the meta tags rendered:
```
    <!-- Twitter -->
    <meta name="twitter:title" content="Democracy Club">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:site" content="@democlub">
    <meta name="twitter:url" content="http://localhost:8000">
    <meta name="twitter:image" content="/media/cache/24/6d/246d0a333710fe425cccd829c8851e68.jpg" />
    <meta name="twitter:description" content=""/>
    <meta name="twitter:image:alt" content="Democracy Club">    
```
I'd have expected the url to be the url of the blog e.g. :
```
    <meta name="twitter:url" content="http://localhost:8000/blog/2021/11/29/hello/">
```
This can be done by adding another `block` in the base.html and then updating the `post_detail.html` to use a combination of the `CANONICAL_URL` and `post.get_absolute_url`

Reply:
Also noticed that the description is empty. Perhaps this should be the title of the blog?